build.gradle

```diff
@@ -0,0 +1,44 @@
+buildscript {
+
+    ext {
+        organization = 'twocoders'
+        groupIdentifier = 'com.twocoders'
+        projectIdentifier = 'utils'
+        projectWebsite = 'https://github.com/Two-Coders/common-extensions'
+        projectDescription = 'Project containing common Android extensions.'
+        projectLicences = ['MIT']
+
+        androidMinSdkVersion = 21
+        androidCompileSdkVersion = 30
+        androidTargerSdkVersion = androidCompileSdkVersion
+
+        libCommonExtensionsVersion = '1.0.0'
+
+        kotlinVersion = '1.4.21'
+        androidXCoreVersion = '1.3.2'
+        annotationVersion = '1.1.0'
+    }
+
+    repositories {
+        google()
+        jcenter()
+
+    }
+
+    dependencies {
+        classpath 'com.novoda:bintray-release:0.9.2'
```

Comment:
btw s novodou je problem pri gradle 4+. Neviem ci to chlapici udrziavaju este... :(

Reply:
Napadlo mě to, když jsem viděl, že jediné co není zvednuté na nejnovější verzi je bintray plugin a gradle :-D budeme to muset do budoucna vyřešit, teď se mi s tím nechtělo zdržovat, mám kopec jiné roboty. Btw. ten nový Gradle Publish task by s Bintray nefungoval? https://developer.android.com/studio/build/maven-publish-plugin

Reply:
podla mna by mohol, ale bolo by sa na to treba pozriet

Reply:
tak jsem na to koukal, ale asi tím že Bintray je celý tak trochu specifický, že používá na autorizaci API key (v tom gradle pluginu je myslím podpora jen pro jméno a heslo), tak to asi nebude tak jednoduché a ten svůj plugin stále udržují, tak nevím. Nanašel jsem k tomu popravdě žádné kloudné info. Ale na posledním projektu jsme to používali s klasickým maven Nexus repozitářem a tam to fungovalo v pohodě.